localisation/hfe_l_english.yml

```diff
@@ -34,53 +34,62 @@
- hfe.47.desc.2:0 "We fought and won our freedom, taking pride in our victory, but pride became arrogance and we gradually forgot our gods. Consumed by the thirst for knowledge, we pushed our ambitions to every corner of the galaxy. It was the Golden Age of our people's civilization, for we unraveled many mysteries of the universe. We visited fascinating worlds and built wonderful cities where life had once been impossible.\nOne thousand years ago, we finally located our ancient homeworld, in the §PSol System,§! and on the beautiful planet §PEarth§! we found the remnants of our kind in a primitive state, and we decided not to disturb them. We haven't visited Sol since then."
+ hfe.47.desc.1:0 "We have not always been like this. To become who we are today, we had to be forged through blood and despair. Millenia ago, our primitive ancestors were kidnapped by a sinister race called the §PAhets,§! to be used for building a colony. Stolen from our homelands and pressed into slavery, we had prepared to resign ourselves to hopelessness. Just when we had given up, we were suddenly freed, and the colony hadn't even been fullly established! Only a short time of our enslavement had passed when §PRa§! Himself, in His blazing chariot, led a rebellion and unleashed His Wrath on our foes! With His help, we destroyed the colonial ship-turned-capitol and slaughtered our former masters.\nThe remaining Ahet survivors were put to labor and our people utilized what was left of their technology to progress our civilization. So our journey began."
+ hfe.47.desc.2:0 "We fought and won our freedom, taking pride in our victory, but pride became arrogance and we gradually forgot our gods. Consumed by the thirst for knowledge, we pushed our ambitions to every corner of the galaxy. It was the Golden Age of our people's civilization, for we unraveled many mysteries of the universe. We visited fascinating worlds and built wonderful cities where life had once been impossible.\nOne thousand years ago, we finally located our ancient homeworld in the §PSolar System.§! On the beautiful planet §PEarth§! we found the remnants of our kind in a primitive state, and we decided not to disturb them. We haven't visited Sol since then."
  hfe.47.desc.3.0.0.0:0 "This is sad news."
  hfe.47.desc.3.0.0.1:0 "This is bad news. Pehaps the §PGreat Pharaoh§! needs to step in."
  hfe.47.desc.3.0.1:0 "However, back to the narrative.\n"
- hfe.47.desc.3.1:0 "Soon, the Gods put us to another trial. We faced a great foe, a detestable race we dubbed the Cylons, but whom you probably know as the Daleks. Our great nation stood up to defend the Galaxy. In the end, we defeated and purged these creatures from everywhere under the light of §PRa.§!"
```

Comment:
Dont kill reference

Reply:
No comment. I always found that reference out of place.

Reply:
Okay, I added it back.